README.md

```diff
@@ -33,6 +33,20 @@ With stylelint, it's easy to start linting your CSS:
 4. Decide whether to use the [CLI](/docs/user-guide/cli.md), [Node API](/docs/user-guide/node-api.md), or [PostCSS plugin](/docs/user-guide/postcss-plugin.md). Be sure to [specify the syntax](/docs/user-guide/css-processors.md#parsing-scss) if you're using SCSS.
 5. Lint!
 
+## Help out
+
+There is always a lot of work to do, and already well over 100 rules to maintain. So please help out in any way that you can!
+
+You can help stylelint grow in a number of ways:
+
+- Create, enhance, and debug rules (see our guide to ["Working on rules"](docs/developer-guide/rules.md)).
+- Improve documentation.
+- Chime in on any open issue or pull request.
+- Open new issues about your ideas for making stylelint better.
```

Comment:
Can we expand on this and encourage experimentation within forks/PRs i.e. rather than just "I have an idea", we encourage "I have an idea and this is how it might work"

Perhaps:

"Open new issues about your ideas for making stylelint better, create a fork and see how it works".